pennylane/tape/tapes/cv_param_shift.py

```diff
@@ -235,19 +235,20 @@ def parameter_shift_first_order(
         op = self._par_info[t_idx]["op"]
         p_idx = self._par_info[t_idx]["p_idx"]
 
-        recipe = op.grad_recipe[p_idx]
-        c, s = (0.5, np.pi / 2) if recipe is None else recipe
-
+        param_shift = op.get_parameter_shift(p_idx)

```

Comment:
Do we also need to update `qubit_param_shift`?

Reply:
That is covered in #915.

Reply:
I've backported this file change to this PR (where it makes more sense)